From cb9a3a0e4144d4fc550877fea8c4c0518fa6c46b Mon Sep 17 00:00:00 2001
From: Owen W. Taylor <otaylor@fishsoup.net>
Date: Tue, 30 Nov 2010 22:22:28 -0500
Subject: [PATCH] autorun: Handle rename of org.gnome.media-handling

When the org.gnome.media-handling schemas was moved to
gsettings-desktop-schemas, it was renamed to
org.gnome.desktop.media-handling.
---
 configure.ac                              |    2 +-
 plugins/automount/gsd-automount-manager.c |    2 +-
 plugins/automount/nautilus-autorun.c      |    4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8b3d004..787714a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -66,7 +66,7 @@ PKG_CHECK_MODULES(SETTINGS_DAEMON,
         gmodule-2.0
         gthread-2.0
         dbus-glib-1 >= $DBUS_GLIB_REQUIRED_VERSION
-	gsettings-desktop-schemas >= 0.1.1
+	gsettings-desktop-schemas >= 0.1.2
 )
 
 PKG_CHECK_MODULES(SETTINGS_PLUGIN,
diff --git a/plugins/automount/gsd-automount-manager.c b/plugins/automount/gsd-automount-manager.c
index d3a7e93..9836dff 100644
--- a/plugins/automount/gsd-automount-manager.c
+++ b/plugins/automount/gsd-automount-manager.c
@@ -217,7 +217,7 @@ gsd_automount_manager_start (GsdAutomountManager *manager,
         g_debug ("Starting automounting manager");
         gnome_settings_profile_start (NULL);
 
-        manager->priv->settings = g_settings_new ("org.gnome.media-handling");
+        manager->priv->settings = g_settings_new ("org.gnome.desktop.media-handling");
         setup_automounter (manager);
 
         gnome_settings_profile_end (NULL);
diff --git a/plugins/automount/nautilus-autorun.c b/plugins/automount/nautilus-autorun.c
index eed336d..9d44c53 100644
--- a/plugins/automount/nautilus-autorun.c
+++ b/plugins/automount/nautilus-autorun.c
@@ -196,7 +196,7 @@ nautilus_autorun_get_preferences (const char *x_content_type,
 	g_return_if_fail (pref_ignore != NULL);
 	g_return_if_fail (pref_open_folder != NULL);
 
-        settings = g_settings_new ("org.gnome.media-handling");
+        settings = g_settings_new ("org.gnome.desktop.media-handling");
 
 	*pref_start_app = FALSE;
 	*pref_ignore = FALSE;
@@ -277,7 +277,7 @@ nautilus_autorun_set_preferences (const char *x_content_type,
 
 	g_assert (x_content_type != NULL);
 
-	settings = g_settings_new ("org.gnome.media-handling");
+	settings = g_settings_new ("org.gnome.desktop.media-handling");
 
 	x_content_start_app = g_settings_get_strv (settings, "autorun-x-content-start-app");
 	x_content_ignore = g_settings_get_strv (settings, "autorun-x-content-ignore");
-- 
1.7.3.2

